Resources/Prototypes/Traits/disabilities.yml

```diff
@@ -55,9 +55,9 @@
     - type: Muted
 
 - type: trait
-  id: WheelchairBound
-  name: trait-wheelchair-bound-name
-  description: trait-wheelchair-bound-desc
+  id: WheelchairUser
```

Comment:
if you change the prototype ID it's going to mess up the db and change deselect it from all of the characters. you should just leave the ID the same for now.